sympy/functions/special/delta_functions.py

```diff
@@ -132,6 +132,12 @@ def fdiff(self, argindex=1):
         >>> diff(DiracDelta(x, 1)).fdiff()
         DiracDelta(x, 3)
 
+        Parameters
+        ==========
+
+        argindex : integer
+                degree of derivative
```

Comment:
I'm not sure how strict Sphinx is over indentation but this should probably either line up with the line above or be indented 4 spaces.

Reply:
I have indented it now.